tests/end_to_end/test_landing_and_sign_in_pages.py

```diff
@@ -1,13 +1,12 @@
 import os
 import re
 
-import pytest
 from playwright.sync_api import expect
 
 
-def test_landing_page(end_to_end_context):
+def test_landing_page(browser):
```

Comment:
This doesn't still need to be `end_to_end_context`, even for the landing page?  I thought it would have to be due to the HTTP basic auth still in place.